quality_of_service_demo/README.md

```diff
@@ -0,0 +1,57 @@
+# Quality of Service Demos
+
+The demo applications in this package show the various Quality of Service settings available in ROS2. The intent is to provide a dedicated application for each QoS type, in order to highlight that single feature in isolation.
+
+## History
```

Comment:
```suggestion
## History

```

See:

> [.md, .rst only] Each section title should be preceded by one empty line and succeeded by one empty line.

-- https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#id7

Reply:
Note, if you use the webui to take the suggestion you have to add your sign-off to the message or it will fail DCO.